### ISSUE TITLE
Can't backup file

### ISSUE THREAD

**Issue by Balloonbear** [rbino/ems-qart#11]
Good evening!

My name is Egor and I'm win10 use. Today I've lost all my tracks during the process of backup via your app. As still, I have no idea what's wrong. Maybe the app is needed an update becasue of new win10 version or something else. Because before everything was pretty good (before re-install of win10). I'm still using the same wire.
When it reads the card's sav it's okay, but then you try to open it via gb emulating tool and it starts to check the save like it's the new one. And somehow, save file from my card is deleted too. Please do something, at least with the backups.

**Comment by rbino** [comment]
Hi,
when reading the SAV, the application just reads (and doesn't write), so while it is possible that the SAV is not read correctly, I find it highly unlikely that ems-qart destroys the SAV from the cart.

Try this: do you have an older (and working) backup? If you do, first do a backup of the SAV in the cart (just in case, even it it doesn't seem to work), then try loading the older SAV on the cart and reading it again and check if the SAV is valid in an emulator. If it is not then there's probably a problem with the cart.

I suppose you're using the last version (0.9.4) since I didn't release a new version since quite a while.

**Comment by Balloonbear** [comment]
Yes, I guess you were right, cuz I've read the data from another card with no problem. But could be possible that the save file itself in the card could be damaged or the problem could only be on the card?

**Comment by rbino** [comment]
The procedure I described in my previous comment should rule out the damage of the SAV file: if you have an older backup and you can succesfully write it and read it again as valid then the cart could be ok. If a valid SAV is broken when written to or read from the cart, then the cart is probably KO.
